dist/debian/control.template

```diff
@@ -16,7 +16,7 @@ Replaces: %{product}-server (<< 1.1)
 Conflicts: %{product}-server (<< 1.1)
 
 Package: %{product}-server
-Architecture: amd64
+Architecture: %{arch}
```

Comment:
I think you can write `any` here: https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-architecture